tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -321,13 +458,24 @@ platform_tests/api/test_psu.py::TestPsuApi::test_power:
     reason: "Unsupported platform API"
     conditions:
       - "asic_type in ['mellanox', 'cisco-8000'] or (asic_type in ['barefoot'] and hwsku in ['newport']) or platform in ['armhf-nokia_ixs7215_52x-r0']"
-
+  xfail:
+    reason: "Testcase consistently fails, raised issue to track"
+      - "hwsku in ['Celestica-DX010-C32']"
```

Comment:
@assrinivasan it should have `conditions:` here. PR test all failed because of invalid yaml file.

Reply:
Fixed in latest commit